src/components/ListMessages.tsx

```diff
@@ -0,0 +1,51 @@
+import { MessageResponse } from "@frequency-chain/api-augment/interfaces";
+import { Button, Space, Table } from "antd";
+import Column from "antd/lib/table/Column";
+import React from "react";
+import { fetchMessagesForSchema } from "../services/chain/apis/extrinsic";
+import { MessageDetails, SchemaProps } from "../services/types";
+import { staticSchema } from "./RegisterSchema";
+
+const ListMessages = (props: SchemaProps): JSX.Element => {
+
+    const [listOfMessages, setListOfMessage] = React.useState<MessageDetails[]>([]);
+
+    const listMessages = async () => {
+        const messages: MessageResponse[] = await fetchMessagesForSchema(parseInt(props.schema.schema_id));
+
+        let allMessages: MessageDetails[] = messages.map((msg) => {
+            return {
+                key: msg.block_number.toString(),
```

Comment:
If you need this key, it doesn't work. Key needs to be unique and block_number is not (well it is only on instant sealing

You likely want to use the index of the map function to give it a unique key number.